docs/next/modules/en/pages/reference/CAPRKE2.adoc

```diff
@@ -0,0 +1,11 @@
+= Cluster API Provider RKE2
+
+== What is Cluster API Provider RKE2?
+
+The link:https://cluster-api.sigs.k8s.io/[Cluster API] brings declarative, Kubernetes-style APIs to cluster creation, configuration and management.
+
+Cluster API Provider RKE2 (CAPRKE2) is a combination of 2 provider types, a Cluster API Control Plane Provider for provisioning Kubernetes control plane nodes and a Cluster API Bootstrap Provider for bootstrapping Kubernetes on a machine where link:https://docs.rke2.io/[RKE2] is used as the Kubernetes distro.
+
+== Learn more
+
+To learn more about CAPRKE2, visit its dedicated documentation link:https://caprke2.docs.rancher.com/[website].
```

Comment:
This is a copy-paste of the introductory paragraph on CAPRKE2 website. I am not sure how much we want to add to the Turtles docs in terms of "general overview of CAPRKE2". Hence, the PR is a Draft.

@rancher/highlander please suggest what would be good to be added here.